includes/gateway/class-omise-payment-creditcard.php

```diff
@@ -642,9 +642,25 @@ public function get_icon() {
 		public function omise_scripts() {
 			if ( is_checkout() && $this->is_available() ) {
 				wp_enqueue_script( 'omise-js', 'https://cdn.omise.co/omise.js', array( 'jquery' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
-				wp_enqueue_script( 'omise-util', plugins_url( '../../assets/javascripts/omise-util.js', __FILE__ ), array( 'omise-js' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
-				wp_enqueue_script( 'omise-payment-form-handler', plugins_url( '../../assets/javascripts/omise-payment-form-handler.js', __FILE__ ), array( 'omise-js', 'omise-util' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
-				wp_localize_script( 'omise-payment-form-handler', 'omise_params', array( 'key' => $this->public_key() ) );
+				wp_enqueue_script( 'omise-payment-form-handler', plugins_url( '../../assets/javascripts/omise-payment-form-handler.js', __FILE__ ), array( 'omise-js' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
+
+				$omise_params = array(
+					'key'                            => $this->public_key(),
+					'required_card_name'             => __( 'Card Holder\'s name is a required field', 'omise' ),
+					'required_card_number'           => __( 'Card Number is a required field', 'omise' ),
+					'required_card_expiration_month' => __( 'Card Expiry month is a required field', 'omise' ),
```

Comment:
Same as before - the word 'Card' is superfluous in this and the following 2 field messages

Reply:
Also, inconsistent capitalisation of field name words. Some have all words capitalised, some don't. Need to pick a style and stay with it